pybossa/cache/project_stats.py

```diff
@@ -175,33 +175,20 @@ def stats_dates(project_id, period='15 day'):
 
     params = dict(project_id=project_id, period=period)
 
-    # Get all completed tasks
+    # Get all tasks with at least 1 task run and its last finish time
     sql = text('''
-               WITH myquery AS (
```

Comment:
Do we have an associated unit test to confirm the changes of this query still perform the same functionality of the original?

Reply:
Yes, I have the [commit](https://github.com/bloomberg/pybossa/pull/680/commits/87de10e266e90254c9a122844dfe02d78b08ee46) to enhance the unit test and tested locally to make sure the behave the same.